docs/source/outputs.rst

```diff
@@ -0,0 +1,103 @@
+Simulator Outputs
+=================
+
+This page describes the outputs generated by the simulator. All output files are placed relative to the directory where the simulation is executed. This section explains the content and purpose of each file or folder created by the simulator.
+
+Base Directory
+--------------
+
+After a successful run, the base directory (i.e., where the simulation was launched) will contain the following files:
+
+- **Simulation log file**:  
+  A file named ``pydamus_<date>_<time>.log`` is created during the simulation. It contains runtime information, including progress updates, warnings, and error messages (if any). This log is essential for debugging and performance diagnostics.
+
+- **Completion flag**:  
+  If the simulation completes successfully, an empty file named ``simulation_config.json.SUCCESS`` is written. This is used by the Blue Brain Project (BBP) workflow system to detect successful execution. The presence of this file indicates the run ended without errors.
+
+output/
+-------
+
+This folder contains the simulation results, including all reports requested in ``simulation_config.json``. The exact location of the ``output`` folder is specified within the configuration file.
+
+Contents:
+
+- **Reports**:  
+  All the time series or scalar reports explicitly configured in ``simulation_config.json`` are saved here. The format and structure depend on the report type and backend.
+
+- **Spike files**:  
+  The final spike output is written in two formats:
+  
+  - ``out.dat``: Plain text file listing spike events (only present when using CoreNEURON)
+  - ``out.h5``: HDF5 version of the same data for efficient access
```

Comment:
Great to have this succinct info here; maybe we can link also to the official SONATA description for the h5 format?
https://github.com/AllenInstitute/sonata/blob/master/docs/SONATA_DEVELOPER_GUIDE.md#spike-file